components/omega/src/ocn/EosConstants.h

```diff
@@ -0,0 +1,92 @@
+#ifndef OMEGA_EOSCONSTANTS_H
+#define OMEGA_EOSCONSTANTS_H
+//===-- ocn/Eos.h - Equation of State --------------------*- C++ -*-===//
+//
+/// \file
+/// \brief Contains functors for calculating specific volume
+///
+/// This header defines functors to be called by the time-stepping scheme
+/// to calculate the specific volume based on the choice of EOS
+//
+//===----------------------------------------------------------------------===//
+
+namespace OMEGA {
+
+#define GSW_SPECVOL_COEFFICIENTS
+const double V000 = 1.0769995862e-03;
```

Comment:
These should all also be constexpr rather than const so the compiler can treat them as compile-time constants.